### PR TITLE
Regenerate client

### PR DESCRIPTION
**Pull request by agourlay**
Regenerate client to sync. our dev package with the latest changes in the API.

### CODE REVIEW

File: qdrant_client/http/models/models.py [hunk 1]
```diff
@@ -1031,7 +1079,10 @@ class OptimizersConfig(BaseModel, extra="forbid"):
         description="Maximum size (in kilobytes) of vectors allowed for plain index, exceeding this threshold will enable vector indexing  Default value is 20,000, based on &lt;https://github.com/google-research/google-research/blob/master/scann/docs/algorithms.md&gt;.  To disable vector indexing, set to `0`.  Note: 1kB = 1 vector of size 256.",
     )
     flush_interval_sec: int = Field(..., description="Minimum interval between forced flushes.")
-    max_optimization_threads: int = Field(..., description="Maximum available threads for optimization workers")
+    max_optimization_threads: Optional[int] = Field(
```
Comment (agourlay) [inline qdrant_client/http/models/models.py:1082]:
```
    __pydantic_self__.__pydantic_validator__.validate_python(data, self_instance=__pydantic_self__)
pydantic_core._pydantic_core.ValidationError: 1 validation error for OptimizersConfig
max_optimization_threads
  Input should be a valid integer [type=int_type, input_value=None, input_type=NoneType]
  ```

**Comment by joein** [comment]
I dared to rebase to make ci green
not sure if we need to merge it now or closer to the release though

File: qdrant_client/http/models/models.py (hunk 1 above)
Comment (semoal) [reply qdrant_client/http/models/models.py:1082]: We're getting this error on `qdrant-client = "^1.7.3"`;

Is it something expected?

Comment (joein) [reply qdrant_client/http/models/models.py:1082]: do you use dev build?

Comment (semoal) [reply qdrant_client/http/models/models.py:1082]: Ah yess, that's the issue; 

ARG QDRANT_VERSION=dev

FROM qdrant/qdrant:${QDRANT_VERSION}

Which one should I use with 1.7.3 client?

Comment (joein) [reply qdrant_client/http/models/models.py:1082]: latest (1.7.4)

Comment (joein) [reply qdrant_client/http/models/models.py:1082]: we have some backward compatibility issue (dev is not backward compatible atm), but we have already made required fixes (but haven't merged them yet)

**Comment by joein** [comment]
This PR played its important role by giving us a hope for a better future (it was green) when everything was falling apart (ci was red).
I am closing it due to the conflicts now
F. :saluting_face:

**Pull request closed without merging**